Documentation/metrics.md

```diff
@@ -18,7 +18,7 @@ Metrics that are etcd2 related are documented [v2 metrics guide][v2-http-metrics
 
 These metrics describe the status of the etcd server. In order to detect outages or problems for troubleshooting, the server metrics of every production etcd cluster should be closely monitored.
 
-All these metrics are prefixed with `etcd_server_`
+All these metrics are prefixed with `etcd_server_`, except `lease_expired_total` with `etcd_debugging_server_`
```

Comment:
we do not really document debugging metrics in this doc. probably we should have a separate section for debugging metrics if we wish?